satpy/etc/composites/abi.yaml

```diff
@@ -147,3 +147,261 @@ composites:
+    ## adapted from etc/composites/viirs.yaml
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C07
+    - name: C06
+    - name: C05
+    standard_name: fire_temperature
+    name: fire_temperature_awips
+
+  land_cloud_fire:
+    ## RGB Day Land Cloud Fire recipe source: http://rammb.cira.colostate.edu/training/visit/quick_guides/QuickGuide_GOESR_DayLandCloudFireRGB_final.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C06
+    - name: C03
+      modifiers: [sunz_corrected]
+    - name: C02
+      modifiers: [sunz_corrected]
+    standard_name: land_cloud_fire
+
+  land_cloud:
+    ## RGB Day Land Cloud Fire recipe source: http://rammb.cira.colostate.edu/training/visit/quick_guides/QuickGuide_GOESR_daylandcloudRGB_final.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C05
+    - name: C03
+      modifiers: [sunz_corrected]
+    - name: C02
+      modifiers: [sunz_corrected]
+    standard_name: land_cloud
+
+  day_microphysics_eum:
+    ## adapted from etc/composites/ahi.yaml
+    ## source: http://www.eumetrain.org/rgb_quick_guides/quick_guides/DaymicroRGB.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C03
+      modifiers: [sunz_corrected]
+    - name: C07
+      modifiers: [nir_reflectance]
+    - name: C14
+    standard_name: day_microphysics
+
+  snow:
+    ## adapted from etc/composites/visir.yaml
+    ## source: http://www.eumetrain.org/rgb_quick_guides/quick_guides/SnowRGB.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C03
+      modifiers: [sunz_corrected]
+    - name: C05
+      modifiers: [sunz_corrected]
+    - name: C07
+      modifiers: [nir_reflectance]
+    standard_name: snow
+
+  day_microphysics_abi:
+    ## adapted from etc/composites/ahi.yaml
+    ## Variations to the Day Microphysics RGB product using the 2.3 micron channel
+    ## source: pages 12 and 13 http://www.virtuallab.bom.gov.au/files/3114/7884/4308/NewRGBProductsNov2016RFGmeeting.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C03
+    - name: C06
+    - name: C13
+    standard_name: day_microphysics_ahi
+
+  cloud_phase:
+    ## Day Cloud Phase RGB (name from EUMETSAT)
+    ## it uses the default used in etc/enhancements/generic.yaml of natural_color_default
+    ## source: http://www.eumetrain.org/rgb_quick_guides/quick_guides/CloudPhaseRGB.pdf
+    ## source: https://www.eumetsat.int/website/home/Images/ImageLibrary/DAT_2861499.html
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C05
+      modifiers: [sunz_corrected]
+    - name: C06
+      modifiers: [sunz_corrected]
+    - name: C02
+      modifiers: [sunz_corrected, rayleigh_corrected]
+    standard_name: natural
```

Comment:
The `natural` composite is not consistently called `natural_color` in Satpy. Some sensors had `natural` some had `natural_color`, this was fixed a couple versions ago.

Reply:
It's ready